coderd/database/dbgen/dbgen.go

```diff
@@ -525,3 +525,40 @@ func must[V any](v V, err error) V {
 	}
 	return v
 }
+
+func takeFirstIP(values ...net.IPNet) net.IPNet {
```

Comment:
I think this came from a commit from a different PR/branch you were working on

Reply:
Yeah, I gotta merge in main